frontend/src/layout/TopContent/topContentLogic.ts

```diff
@@ -21,7 +21,7 @@ export const topContentLogic = kea<topContentLogicType<BackTo>>({
         ],
     },
     listeners: ({ actions }) => ({
-        [router.actions.locationChanged]: ({ hashParams }) => {
+        [router.actionTypes.locationChanged]: ({ hashParams }) => {
```

Comment:
what is actiontypes instead of actions?

Reply:
Point 3 in the ["rough edges" section here](https://kea.js.org/docs/guide/typescript#rough-edges) :).